test/models/project_test.rb

```diff
@@ -543,4 +543,24 @@ class ProjectTest < ActiveSupport::TestCase
       _(project.searchable_vector[:d]).must_equal project.description
     end
   end
+
+  describe 'important projects' do
+    it 'should return important code locations project' do
+      enlist = create(:enlistment, project: project)
+      fis_code_location = FisCodeLocation.first
```

Comment:
Change the above to :
```
enlistment = create(:enlistment, project: project)
fis_code_location = enlistment.fis_code_locations.last
```

Rename the variable elsewhere. Same change for the other test.